example/scip_with_jump_minlp.jl

```diff
@@ -0,0 +1,46 @@
+using JuMP
+#using JuMP:MOI
+using SCIP
+
+function solve_model(m::Model;mixed::Bool = false)
+  if mixed
+    @variable(m, x>=0, Int)
+    @variable(m, y[1:2], Int)
+    @variable(m, s[1:5]>=0, Int)
+    @variable(m, l[1:5]>=0, Int)
+  else
+    @variable(m, x>=0)
+    @variable(m, y[1:2])
+    @variable(m, s[1:5]>=0)
+    @variable(m, l[1:5]>=0)
+  end
+  
+  @objective(m, Min, -x -3y[1] + 2y[2])
+
+  @constraint(m, -2x +  y[1] + 4y[2] + s[1] ==  16)
+  @constraint(m,  8x + 3y[1] - 2y[2] + s[2] ==  48)
+  @constraint(m, -2x +  y[1] - 3y[2] + s[3] == -12)
+  @constraint(m,       -y[1] + s[4] ==   0)
+  @constraint(m,        y[1] + s[5] ==   4)
+  @constraint(m, -1 + l[1] + 3l[2] +  l[3] - l[4] + l[5] == 0)
+  @constraint(m,     4l[2] - 2l[2] - 3l[3]               == 0)
+  for i in 1:5
+    @NLconstraint(m, l[i] * s[i] == 0)
```

Comment:
Since this is quadratic, it actually doesn't need the `@NLconstraint`, and could use `@constraint` instead. But it should not matter much to SCIP.

Reply:
Yes, I just wanted to know if @NLcontraint works well with SCIP.
The example code refers to Chap8 in "Julia Programming for Operations Research 2 / e".

Reply:
There are potential copyright issues with copying code verbatim from a book. Please clarify the license of this code.

Reply:
OK~ 
I'll check!

Reply:
I'm not sure what copyright license my codes carry, but I'm good with this. 

Codes were mine, but the actual example is from: 

Ex 9.1.1 from Floudas, C.A., Pardalos, P.M., Adjiman, C., Esposito, W.R., Gümüs, Z.H., Harding, S.T., Klepeis, J.L., Meyer, C.A. and Schweiger, C.A., 2013. Handbook of Test Problems in Local and Global Optimization (Vol. 33). Springer Science & Business Media, http://titan.princeton.edu/TestProblems/ 